workers/unity/Packages/com.improbable.gdk.tools/RedirectedProcess.cs

```diff
@@ -1,13 +1,22 @@
 using System;
+using System.Collections.Generic;
 using System.Diagnostics;
 using System.IO;
 using System.Text;
+using System.Threading.Tasks;
 using Improbable.Gdk.Tools.MiniJSON;
 using UnityEngine;
 using Debug = UnityEngine.Debug;
 
 namespace Improbable.Gdk.Tools
 {
+    public class RedirectedProcessResult
```

Comment:
Nit - I think this might make more sense as a struct, although it makes almost no difference.